doc/connectivity/bluetooth/api/audio/shell/bap.rst

```diff
@@ -217,6 +218,58 @@ ID before syncing to the BIG.
    Sink 0x20019110 is ready to sync without encryption
    uart:~$ bap sync_broadcast 0x01
 
+
+Scan for and establish a broadcast sink stream by broadcast name
+----------------------------------------------------------------
+The command :code:`bap create_sink_by_name` will start scanning and sync to the periodic
```

Comment:
```suggestion
----------------------------------------------------------------

The command :code:`bap create_sink_by_name` will start scanning and sync to the periodic
```

Hmm, actually not sure if the empty line here is needed or not, but we usually do that for section. Could you please add one?